content/docs/rfcs/10/README.md

```diff
@@ -133,6 +133,12 @@ For an experimental privacy-preserving economic spam protection mechanism, see [
 
```

Comment:
Add your name to the contributor list at the top :)

Reply:
resolved in 3c212ea20df7ffb3ee009d1dce5b11fec4537a2c